changelogs/fragments/1615-no_formatted_with_extra_space.yml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+- Fixes to the stack trace output, where it does not contain spaces between each character. The module had incorrectly always outputted extra spaces between each character. (https://github.com/ansible-collections/amazon.aws/pull/1615)`
```

```suggestion
-  'lambda_execute - Fixes to the stack trace output, where it does not contain spaces between each character. The module had incorrectly always outputted extra spaces between each character. (https://github.com/ansible-collections/amazon.aws/pull/1615)'
```

Since this only impacts one module, can you please include the module name in the changelog?